core/src/test/java/org/apache/accumulo/core/iterators/DefaultIteratorEnvironment.java

```diff
@@ -18,17 +18,16 @@
 
 import java.io.IOException;
 
+import org.apache.accumulo.core.client.impl.TestIteratorEnvironment;
 import org.apache.accumulo.core.conf.AccumuloConfiguration;
 import org.apache.accumulo.core.data.Key;
 import org.apache.accumulo.core.data.Value;
-import org.apache.accumulo.core.iterators.IteratorUtil.IteratorScope;
 import org.apache.accumulo.core.iterators.system.MapFileIterator;
-import org.apache.accumulo.core.security.Authorizations;
 import org.apache.accumulo.core.util.CachedConfiguration;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.FileSystem;
 
-public class DefaultIteratorEnvironment implements IteratorEnvironment {
+public class DefaultIteratorEnvironment extends TestIteratorEnvironment {
```

Comment:
I take another look at what I did w/ TestIteratorEnvironment.   I remember being down in the weeds and noticing a lot of redundant implementations of IteratorEnv.  I did something about it, but maybe I went from bad to peculiar.